resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterSchedulerBackend.scala

```diff
@@ -226,6 +267,12 @@ private[spark] class KubernetesClusterSchedulerBackend(
 
   override def start(): Unit = {
     super.start()
+
+    Seq((STATUS_JOB_STATE, JobState.RUNNING),
+      (STATUS_DRIVER_UI, "http://localhost:8001/api/v1/namespaces" +
```

Comment:
does this assume the user is accessing the SparkUI via a `kubectl` proxy on `localhost:8001`?  Can we support ingress-based exposure of the Spark UI too?